Abbyy.CloudSdk.V2.Client/Models/RequestParams/DocumentProcessingParams.cs

```diff
@@ -94,5 +94,30 @@ public sealed class DocumentProcessingParams
 		/// to the result file.
 		/// </summary>
 		public bool? ReadBarcodes { get; set; }
+
+		/// <summary>
+		/// Optional. Default is <see cref="Enums.ImageSource.Auto"/>. Specifies the source of the image.
+		/// </summary>
+		public ImageSource? ImageSource { get; set; }
```

Comment:
Why we have different properties order in DocumentProcessingParams.cs / ImageProcessingParams.cs ? 
Maybe we make same order in code files and on documentation page?

Reply:
Decided to make refactoring in separate PR.